lib/github-glue.ts

```diff
@@ -52,7 +52,7 @@ export class GitHubGlue {
         const [, short, completedAt] = match;
         const url = `https://github.com/git/git/commit/${gitGitCommit}`;
 
-        await this.ensureAuthenticated();
+        await this.ensureAuthenticated(this.authenticated || "gitgitgadget");
```

Comment:
Not clear why `this.authenticated` would be used.  Isn't this an update to the gitgitgadget repo?  Or, for testing, should `owner` in the json use this.authenticated?

Reply:
You're right. Will fix.